api/src/feeds/impl/feeds_api_impl.py

```diff
@@ -125,15 +125,35 @@ def _get_gtfs_feeds(
         bounding_latitudes: str = None,
         bounding_longitudes: str = None,
         bounding_filter_method: str = None,
+        order_by: list[str] = None,
     ) -> List[GtfsFeed]:
+        def _get_order_by_key(order_by: list[str] = None):
+            order_by_columns = []
+            for field in order_by:
+                if "country_code" in field:
```

Comment:
approving, but wondering if we shouldn't just make it `if field == "-country_code"` and so on